src/ONE.Models.CSharp/Common/Configuration/Spreadsheet/GroupHeader.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections.Generic;
+using ONE.Models.CSharp.Enums;
+
+namespace ONE.Models.CSharp.Models.Common.Configuration.Spreadsheet
+{
+    public class GroupHeader
```

Comment:
It seems like most of these classes are basically an effort to strongly type json data.  Perhaps the namespace/naming convention should reflect that.
Additionally it has been suggested that this sort of thing should be added to the protobuf models so that we don't have everyone writing their own POCO to translate the same json data.